specs/Modal.spec.js

```diff
@@ -253,18 +255,33 @@ export default () => {
     (document.body.className.indexOf("custom-modal-open") > -1).should.be.ok();
   });
 
-  it("don't append class to document.body if modal is not open", () => {
+  it("supports setting react modal open class in <html />.", () => {
+    renderModal({ isOpen: true, htmlOpenClassName: "custom-modal-open" });
+    isHtmlWithReactModalOpenClass("custom-modal-open").should.be.ok();
+  });
+
+  // eslint-disable-next-line max-len
+  it("don't append class to document.body if modal is closed.", () => {
     renderModal({ isOpen: false });
     isBodyWithReactModalOpenClass().should.not.be.ok();
-    unmountModal();
   });
 
-  it("append class to document.body if modal is open", () => {
+  it("don't append class to <html /> if modal is closed.", () => {
+    renderModal({ isOpen: false, htmlOpenClassName: "custom-modal-open" });
+    isHtmlWithReactModalOpenClass().should.not.be.ok();
+  });
+
+  it("append class to document.body if modal is open.", () => {
     renderModal({ isOpen: true });
     isBodyWithReactModalOpenClass().should.be.ok();
-    unmountModal();
   });
 
+  it("don't append class to <html /> if not defined.", () => {
```

Comment:
Here's the test that I mentioned above about the HTML class not being applied by default.

Reply:
Yes, that makes sense, thanks for clarifying!

Reply:
It's more of a conservative approach, but we can later change to define it by default.

Reply:
Hey @diasbruno, shouldn't doc mention that you have to explicitly define `htmlOpenClassName` for it to work?

Reply:
@woniesong92 I updated the docs to mention that this prop is `null` by default. I'll check if the published doc is not updated, but I'm sure I publish `3.3.1` and the new docs.